app/controllers/addresses_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AddressesController < ApplicationController
+    def create
+        @address = Address.new(address_params)
+        if @address.save
+            render json: {product: @address, message: "Your product is added"}
+        else
+            render json: {product: @address.errors.messages, message: "Sorry you address is not added"}
```

Comment:
No need to send messages two time
render jason: {errors:  address.errors.messages}